recipe/meta.yaml

```diff
@@ -22,16 +22,26 @@ requirements:
   build:
     - toolchain
     - python
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
     - cmake
     - boost 1.62.*
     - mpfr
     - zlib 1.2.*
     - eigen
     - swig
+    - mpir  # [win]
+    - gmp  # [unix]
```

Comment:
But isn't that "too implicit" ? I like going explicit on dependencies, but it's your call.